### PR TITLE
Update shared.lua

### PR DESCRIPTION
**Pull request by RubberDuck1977**
i hope this is the correct way now
added soil,glass,sand,pure sand and capacitor 1

### CODE REVIEW

**Comment by sachera** [comment]
It is.

Could you please include the changes regarding the info.json and shared.jua into a single pull request. Pull request should contain all changes needed to sole a certain issue or to add a feature. Since you add an item in this pull request from the mod pyrawores the change in info.json to include this mod as an optional dependency should therefore be part of the same (this) pull request.

**Comment by RubberDuck1977** [comment]
o and how do i do that?
im a noob so forgive me

**Comment by RubberDuck1977** [comment]
> o and how do i do that?
> im a noob so forgive me
i did spot a mistake in the coding already
also i could not get capacitor 1 stacked

**Comment by sachera** [comment]
I would cherry pick the commit from the other branch. but it is probably easier for the beginning to just reapply the same changes on the branch this pull request was made from (https://github.com/RubberDuck1977/factorio-deadlocks-stacking-for-pyanodon/tree/patch-2). The pull request should be updated automatically as soon as you commit and push these changes to the patch-2 branch.

File: src/migrations/shared.lua [hunk 1]
```diff
@@ -32,8 +35,11 @@ shared.STACKABLES = {
     { item = "processing-unit", icon = nil, stage = 3 },
     { item = "graphite", icon = nil, stage = 1 },
     { item = "fiberboard", icon = nil, stage = 1 },
-    { item = "pcb1", icon = nil, stage = 1 }
-  }
+    { item = "pcb1", icon = nil, stage = 1 },
+    { item = "capacitor-1", icon = nil, stage = 1 }
```
Comment (sachera) [inline src/migrations/shared.lua]: its called "capacitor1" without the "-"
```suggestion
     { item = "capacitor1", icon = nil, stage = 1 }
```

File: src/migrations/shared.lua [hunk 2]
```diff
@@ -32,8 +35,11 @@ shared.STACKABLES = {
...
+  },
+ pyrawores = {
+    { item = "glass", icon = nil, stage = 1 }
```
Comment (sachera) [inline src/migrations/shared.lua:42]: Seems like you forgot to close one brace pair

```suggestion
    { item = "glass", icon = nil, stage = 1 }
   }
```

**Review by sachera — CHANGES_REQUESTED** [review]
Change the marked lines and add the changes to the info.json as requested earlier. As soon as thats done i will merge the pull-request and release the new Version.

**Comment by RubberDuck1977** [comment]
im currently changing the code on your end now

**Pull request closed without merging**